lib/src/component_declaration/component_base_2.dart

```diff
@@ -559,7 +546,6 @@ abstract class UiStatefulComponent2<TProps extends UiProps, TState extends UiSta
 ///
 /// __Initializing state:__
 ///
-///     @Component2()
 ///     class YourComponent extends UiComponent2<YourProps> with UiStatefulMixin2<YourProps, YourState> {
 ///       @override
 ///       void init() {
```

Comment:
Woops... I know this is out-of-scope for this PR... but lets clean this up while we're here.